x-pack/examples/triggers_actions_ui_example/public/components/alerts_filters_form_sandbox.tsx

```diff
@@ -12,11 +12,14 @@ import { AlertsFiltersForm } from '@kbn/response-ops-alerts-filters-form/compone
 import { EuiFlexGroup, EuiFlexItem, EuiLoadingSpinner, EuiPanel, EuiText } from '@elastic/eui';
```

Comment:
Shall we remove the sandbox now that the form is integrated in the panel creation flow?

Reply:
I'd remove it so it doesn't become outdated compared to the real usage

Reply:
I agree! Let's do it!